apps/console/src/app/components/api-keys/ProviderApiKeyListItem.tsx

```diff
@@ -65,15 +65,19 @@ export const ProviderApiKeyListItem = ({
   };
 
   const handleSave = async () => {
-    await updateKeyMutation.mutateAsync({
-      provider,
-      value: editValue,
-      organizationId: currentOrgId,
-    });
+    if (!editValue) {
+      messageApi.error("API key is required");
+    } else {
+      await updateKeyMutation.mutateAsync({
+        provider,
+        value: editValue,
+        organizationId: currentOrgId,
+      });
```

Comment:
Please don't use the `messageApi` for this. We have the ability to render errors under their input field as we do in other places (e.g. prompt creation modal, environment creation modal). Suits better.